src/assets/scss/components/_list-item.scss

```diff
@@ -71,3 +71,25 @@
     }
   }
 }
+
+@keyframes example {
```

Comment:
I think we can think about a better name for it :-)

Reply:
Yes, my mistake, forgot to update the name, now it is good